README.md

```diff
@@ -29,7 +29,7 @@ There are groups of tasks we have prepared for this exercise. Inside the groups
 
 There are purposefully too many tasks, there is no goal to do all of them. Use your judgement working on the tasks. It’s okay to skip a task that is needlessly difficult.
 
-Please do at least one task from every group to show some diversity, though.
+Please do at least one task from every group to show some diversity, though, and more is always better as long as you fit into the time allocated for the exercise.
```

Comment:
I'd like to work the expected time in there, so it's really clear. We also pay for this so we can ask they spend some time on it. 

> We would like you to spend ~8 hours of dev time on this. In this time, please do at least one task from each group to show some diversity, and use the remaining time to do additional tasks from each group. So a minimum of 4 tasks, and more if you have time.

Reply:
Something like the above, but doesn't have to be this exactly.